dts/bindings/adc/nxp,kinetis-adc16.yaml

```diff
@@ -20,8 +20,38 @@ properties:
     interrupts:
       required: true
 
+    dma-channel:
+      type: int
+      required: false
+      description: dma channel to use, for periodic trigger shall be 0-3
+
     "#io-channel-cells":
       const: 1
 
+    clk-source:
+      type: int
+      required: false
+      description: use alternate clock reference source
+
+    long-sample:
+      type: int
+      required: false
+      description: enable long sample mode
```

Comment:
seems like a boolean from the description.  What what the value mean?

Reply:
> seems like a boolean from the description. What what the value mean?

Seems like the description has an issue. From higher up in the driver:

```c
	adc_config.longSampleMode =
		(adc16_long_sample_mode_t)config->long_sample;
```

And in the HAL:

```
typedef enum _adc16_long_sample_mode
{
    kADC16_LongSampleCycle24  = 0U, /*!< 20 extra ADCK cycles, 24 ADCK cycles total. */
    kADC16_LongSampleCycle16  = 1U, /*!< 12 extra ADCK cycles, 16 ADCK cycles total. */
    kADC16_LongSampleCycle10  = 2U, /*!< 6 extra ADCK cycles, 10 ADCK cycles total. */
    kADC16_LongSampleCycle6   = 3U, /*!< 2 extra ADCK cycles, 6 ADCK cycles total. */
    kADC16_LongSampleDisabled = 4U, /*!< Disable the long sample feature. */
} adc16_long_sample_mode_t;
```

So I guess this should really be an enum with those values?

Reply:
Thanks. update to enum